components/com_content/tmpl/category/default_articles.php

```diff
@@ -123,49 +123,48 @@
             <caption class="visually-hidden">
                 <?php echo Text::_('COM_CONTENT_ARTICLES_TABLE_CAPTION'); ?>
             </caption>
-            <?php if ($this->params->get('show_headings')) : ?>
-                <thead>
-                    <tr>
-                        <th scope="col" id="categorylist_header_title">
-                            <?php echo HTMLHelper::_('grid.sort', 'JGLOBAL_TITLE', 'a.title', $listDirn, $listOrder, null, 'asc', '', 'adminForm'); ?>
+            <thead <?php echo $this->params->get('show_headings', '1') ? '' :  'class="visually-hidden"'; ?>>
```

Comment:
Not sure its that important but this code will result in `<thead >`
But the suggested code will produce `<thead>`
```suggestion
            <thead<?php echo $this->params->get('show_headings', '1') ? '' :  ' class="visually-hidden"'; ?>>
```